acrechain/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "acrechain",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Acrechain",
+      "denom_units": [
+        {
+          "denom": "aacre",
+          "exponent": 0
+        },
+        {
+          "denom": "ACRE",
```

Comment:
can you make this lowercase please so it matches the 'display' denom.

Reply:
Okay

Reply:
Done